content/en/docs/tasks/security/authorization/authz-td-migration/index.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Trust domain migration for authorization policy.
+description: Shows how to migrate from one trust domain to another without changing authorization policy.
+weight: 40
+keywords: [security,access-control,rbac,authorization,trust domain, migration]
+---
+
+This task shows you how to migrate from one trust domain to another without changing authorization policy.
+
+In Istio 1.4, we introduce an alpha feature to support {{< gloss >}}trust domain migration{{</ gloss >}} for authorization policy. This means if an
+ Istio mesh needs to change its {{< gloss >}}trust domain{{</ gloss >}}, the authorization policy doesn't need to be changed manually.
+ In Istio, if a {{< gloss >}}workload{{</ gloss >}} is running in namespace `foo` with the service account `bar`, and the trust domain of the system is `my-td`,
+ the identity of said workload is `spiffe://my-td/ns/foo/sa/bar`. By default, the Istio mesh trust domain is `cluster.local`,
+ unless you specify it during the installation.
+
+## Before you begin
+
+1. Read the [authorization concept guide](/docs/concepts/security/#authorization).
+
+1. Install Istio with a custom trust domain and mutual TLS enabled.
+
+    {{< text bash >}}
+    $ cat <<EOF > ./td-installation.yaml
+    apiVersion: install.istio.io/v1alpha2
+    kind: IstioControlPlane
+    spec:
+      values:
+        global:
+          controlPlaneSecurityEnabled: false
+          mtls:
+            enabled: true
+          trustDomain: old-td
+    EOF
+    $ istioctl manifest apply --set profile=demo -f td-installation.yaml
+    {{< /text >}}
+
+1. Follow [Deploy httpbin and sleep](/docs/examples/httpbin-sleep) and:
+
+    * Deploy `httpbin` and `sleep` in the `default` namespace.
+    * Deploy `sleep` in the `sleep-allow` namespace.
```

Comment:
@pitlv2109 You need to remove the example from this PR before we can merge. I suggest changing this step to the following:
```
1. Start the [httpbin]({{< github_tree >}}/samples/httpbin) sample in the
    `default` namesapce and the [sleep]({{< github_tree >}}/samples/sleep)
    sample in the `default` and `sleep-allow` namespaces:

    {{< text bash >}}
    $ kubectl label namespace default istio-injection=enabled
    $ kubectl label namespace sleep-allow istio-injection=enabled
    $ kubectl apply -f @samples/httpbin/httpbin.yaml@
    $ kubectl apply -f @samples/sleep/sleep.yaml@
    $ kubectl apply -f @samples/sleep/sleep.yaml@ -n sleep-allow
    {{< /text >}}
```

Reply:
Done. PTAL.